spec/Section 3 -- Type System.md

```diff
@@ -1570,6 +1624,37 @@ Literal Value            | Variables               | Coerced Value
 `{ b: $var }`            | `{ var: null }`         | Error: {b} must be non-null.
 `{ b: 123, c: "xyz" }`   | `{}`                    | Error: Unexpected field {c}
 
+
+Following are examples of input coercion for a Oneof Input Object with a
+`String` member field `a` and an `Int` member field `b`:
+
+```graphql example
+input ExampleInputTagged @oneOf {
+  a: String
+  b: Int
+}
+```
+
+Literal Value            | Variables               | Coerced Value
+------------------------ | ----------------------- | ---------------------------
+`{ a: "abc", b: 123 }`   | `{}`                    | Error: Exactly one key must be specified
+`{ a: null, b: 123 }`    | `{}`                    | Error: Exactly one key must be specified
+`{ b: 123 }`             | `{}`                    | `{ b: 123 }`
+`{ a: $var, b: 123 }`    | `{ var: null }`         | Error: Exactly one key must be specified
```

Comment:
https://github.com/graphql/graphql-spec/pull/825/files#r583465590

Seems so...  I am asking because until now there has not been a single validation rule requiring the validation of variables, that is, a successful validation of the incoming graphql document clearly spoke of its validity. It was possible to cache this document and not applying all validation rules over and over again. Now this is not the case. Ping @shane32 .

Reply:
Directly above this block is an extremely similar validation block for regular input objects that also factor in the variables. How can you know if a runtime input object is valid if you don't coerce it?

I think this conversation can be resolved, but I'm leaving it open in case I'm missing something that makes what I'm doing with oneof input objects somehow different to what we're doing with regular input objects.

Reply:
> extremely similar validation block for regular input objects that also factor in the variables. 

@shane32 we need to check it

Reply:
I'm going to close this as I don't think any changes are required.